platform/linux-generic/include/odp/api/plat/odp_inline.h

```diff
@@ -0,0 +1,27 @@
+/* Copyright (c) 2018, Linaro Limited
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ */
+
+/* Note: no guardian define here, this header might get included several times
+ * under different conditions */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @cond _ODP_HIDE_FROM_DOXYGEN_ */
+
+#undef _ODP_INLINE
+#ifdef _ODP_API_IMPL
```

Comment:
For example, _ODP_API_NOT_INLINED would be more understandable define name. This define is defined when xxx_inlines.h is used for not inlined implementation of the functions.